.setup/bin/sample_courses/__init__.py

```diff
@@ -42,17 +55,34 @@ def parse_args():
 SETUP_DATA_PATH = os.path.join(CURRENT_PATH, "..", "..", "data")
 
 args = parse_args()
-# Default values, will be overwritten in `main()` if corresponding arguments are supplied
+
 SUBMITTY_INSTALL_DIR = args.install_dir
 SUBMITTY_DATA_DIR = args.data_dir
 SUBMITTY_REPOSITORY = os.path.join(SUBMITTY_INSTALL_DIR, "GIT_CHECKOUT/Submitty")
 MORE_EXAMPLES_DIR = os.path.join(SUBMITTY_INSTALL_DIR, "more_autograding_examples")
 TUTORIAL_DIR = os.path.join(SUBMITTY_INSTALL_DIR, "GIT_CHECKOUT/Tutorial", "examples")
 
-DB_HOST = "localhost"
-DB_PORT = 5432
-DB_USER = "submitty_dbuser"
-DB_PASS = "submitty_dbuser"
+DB_HOST:str = "localhost"
+DB_PORT:int = 5432
+DB_USER:str = "submitty_dbuser"
+DB_PASS:str = "submitty_dbuser"
+
+if not os.path.isdir(SUBMITTY_INSTALL_DIR):
+    raise SystemError(f"The following directory does not exist: {SUBMITTY_INSTALL_DIR}")
+if not os.path.isdir(SUBMITTY_DATA_DIR):
+    raise SystemError(f"The following directory does not exist: {SUBMITTY_DATA_DIR}")
+for directory in ["courses"]:
+    if not os.path.isdir(os.path.join(SUBMITTY_DATA_DIR, directory)):
+        raise SystemError("The following directory does not exist: " + os.path.join(
+            SUBMITTY_DATA_DIR, directory))
+    
+with open(os.path.join(SUBMITTY_INSTALL_DIR, "config", "database.json")) as database_config:
+    database_config_json = json.load(database_config)
+    
+    DB_USER = database_config_json["database_user"]
+    DB_HOST = database_config_json["database_host"]
+    DB_PORT = database_config_json["database_port"]
+    DB_PASS = database_config_json["database_password"]
```

Comment:
I had to move this from main.,py to __init__.py because this was changing the ports value and if you use from filename import variable, the variables sort of become a constant variable that can't be changed
Diff:
https://github.com/Submitty/Submitty/pull/9753/commits/2635f7e0a27d93154b00d74f8b1fdfd448821447